docs/Best Practice Documents/1.2. SDL App Icon Display/index.md

```diff
@@ -0,0 +1,117 @@
+# 1.2. SDL App Icon Display
+
+## 1. Overview
+This chapter describes how the HU displays the SDL App Icon on the screen by the connecting Mobile to the HU.
+
+## 2. Background/Purpose/Reason for Standardization
+Currently, the SDL App Icon display is an SDL standard behavior, however there are some issues the following below:
+
+- The position of SDL App Icon display changes every time it connects the Mobile.
```

Comment:
Yes, that's right. Before executing `updateAppList`, the order of App is decided by makelist (). The order is not fixed because it is the order of AppIDs that are automatically assigned at startup.
We proposed this on the evolutyion repository.